source/admin_manual/list_of_events.rst

```diff
@@ -1290,6 +1290,8 @@ mail_cache_lookup_finished
 
 .. versionadded:: 2.3.15
 
+.. versionremoved:: 2.3.18 Removed for performance reasons.
```

Comment:
Shouldn't the text go indented next line?

Reply:
The end result looks now like this:

`Removed in version 2.3.18: Removed for performance reasons.`

So I think it's ok?

Reply:
ok